app/client/src/PluginActionEditor/components/PluginActionResponse/components/QueryResponseTab/QueryResponseTab.tsx

```
@@ -0,0 +1,480 @@
+                ) : (
+                  actionResponse.body && (
+                    <div data-testid="t--query-error">
+                      {actionResponse.body}
+                    </div>
+                  )
+                ))}
+              <LogHelper
+                logType={LOG_TYPE.ACTION_EXECUTION_ERROR}
+                name="PluginExecutionError"
+                pluginErrorDetails={
+                  actionResponse && actionResponse.pluginErrorDetails
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)